jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/CloseStatus.java

```diff
@@ -171,6 +172,19 @@ public static CloseStatus getCloseStatus(Frame frame)
         return null;
     }
 
+    public static boolean isOrdinary(CloseStatus closeStatus)
```

Comment:
This is kind of underdone method (yes I know I suggested it).   With it we are implementing just a minimal precedence between close stati, so that if we think we are NORMAL, but get a NO_CLOSE, then we are no longer NORMAL and remember the NO_CLOSE... but if we are already PROTOCOL, then we don't care if the other end does a NO_CLOSE... we are still protocol.

So is this `isOrdinary` sufficient... or should we actually just give every status a precedence and change the status only if a higher precedence comes in?      Not sure.... perhaps a TODO comment to consider this would be OK for now,.